Wangscape/noise/RasterBase.h

```diff
@@ -0,0 +1,33 @@
+#pragma once
+#include <vector>
+#include <noise/noise.h>
+#include <SFML/Graphics.hpp>
+#include <functional>
+
+namespace noise
+{
+
+class RasterBase
+{
+public:
+    typedef module::Module NoiseModule;
```

Comment:
I think the `NoiseModule` typedef can just be removed. It was more useful when `Raster*`/`NoiseMap*` were outside of the `noise` namespace and had to use the typename `noise::module::Module`.